### ISSUE TITLE
DecimalUnitType, IntegerUnitType: unit is not accessible for screen readers

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#103]
Unit is not attached to the input nor to the label and it's not read by screen reader.

https://github.com/TomaszGasior/RadioLista-v3/blob/04f199b474292aae521bc4b1b91999d864eb8fbd/templates/common/form/custom_types.html.twig#L3

This field type is used in radio station edit form for these fields: frequency, power, distance from the transmitter, max signal level; also in radio list settings form for custom width field.

#15

**Comment by TomaszGasior** [comment]
Closing because this problem is already fixed in new design. Solution will be available for users when new design will be finished.
